go/test/endtoend/reparent/utils/utils.go

```diff
@@ -258,10 +258,16 @@ func getMysqlConnParam(tablet *cluster.Vttablet) mysql.ConnParams {
 	return connParams
 }
 
-// RunSQLs is used to run SQL commands directly on the MySQL instance of a vttablet
+// RunSQLs is used to run SQL commands directly on the MySQL instance of a vttablet. All commands are
+// run in a single connection.
 func RunSQLs(ctx context.Context, t *testing.T, sqls []string, tablet *cluster.Vttablet) (results []*sqltypes.Result) {
+	tabletParams := getMysqlConnParam(tablet)
+	conn, err := mysql.Connect(ctx, &tabletParams)
+	require.Nil(t, err)
+	defer conn.Close()
+
 	for _, sql := range sqls {
-		result := RunSQL(ctx, t, sql, tablet)
+		result := execute(t, conn, sql)
```

Comment:
The change in this function is that we ensure all queries run in the same conection. This is required for sequences like `[]string{"SET GLOBAL read_only = ON", "FLUSH TABLES WITH READ LOCK", "UNLOCK TABLES"}` above.